ts/features/itwallet/lifecycle/screens/ItwIdentityNotMatchingScreen.tsx

```diff
@@ -26,33 +37,70 @@ export const ItwIdentityNotMatchingScreen = () => {
   };
 
   const handleCancel = () => {
-    dispatch(logoutRequest({ withApiCall: true }));
+    Alert.alert(
+      I18n.t(
+        "features.itWallet.identification.notMatchingIdentityScreen.alert.title"
+      ),
+      I18n.t(
+        "features.itWallet.identification.notMatchingIdentityScreen.alert.message"
+      ),
+      [
+        {
+          text: I18n.t("global.buttons.exit"),
+          style: "destructive",
+          onPress: () => dispatch(logoutRequest({ withApiCall: true }))
+        },
+        {
+          text: I18n.t("global.buttons.cancel"),
+          onPress: constNull // Do nothing, just dismiss the alert
+        }
+      ],
+      { cancelable: false }
+    );
   };
 
   return (
-    <OperationResultScreenContent
-      pictogram="attention"
-      title={I18n.t(
-        "features.itWallet.identification.notMatchingIdentityScreen.title"
-      )}
-      subtitle={I18n.t(
-        "features.itWallet.identification.notMatchingIdentityScreen.subtitle"
-      )}
-      isHeaderVisible={false}
-      action={{
-        label: I18n.t(
-          "features.itWallet.identification.notMatchingIdentityScreen.action"
-        ),
-        accessibilityLabel: I18n.t(
-          "features.itWallet.identification.notMatchingIdentityScreen.action"
-        ),
-        onPress: resetWallet
-      }}
-      secondaryAction={{
-        label: I18n.t("global.buttons.cancel"),
-        accessibilityLabel: I18n.t("global.buttons.cancel"),
-        onPress: handleCancel
-      }}
-    />
+    <>
+      <IOScrollViewWithLargeHeader
+        title={{
+          label: I18n.t(
+            "features.itWallet.identification.notMatchingIdentityScreen.title"
+          )
+        }}
+        headerActionsProp={{ showHelp: true }}
+        goBack={handleCancel}
+      >
+        <ContentWrapper>
+          <VStack space={24}>
+            <IOMarkdown
+              content={I18n.t(
+                "features.itWallet.identification.notMatchingIdentityScreen.message"
+              )}
+            />
+            <Banner
+              content={I18n.t(
+                "features.itWallet.identification.notMatchingIdentityScreen.banner.title"
+              )}
+              pictogramName="security"
+              size="small"
+              color="neutral"
+            />
+          </VStack>
+        </ContentWrapper>
+      </IOScrollViewWithLargeHeader>
+      <FooterActions
+        actions={{
+          type: "TwoButtons",
+          primary: {
+            label: I18n.t("global.buttons.continue"),
+            onPress: resetWallet
+          },
+          secondary: {
+            label: I18n.t("global.buttons.exit"),
+            onPress: handleCancel
+          }
+        }}
+      />
+    </>
```

Comment:
Would it be possible to use the `actions` prop of the `IOScrollViewWithLargeHeader` component instead of `FooterActions`?

Reply:
Right, thank you for the suggestion. Addressed in e2af39633aa04db78ce854a7cdef42b58216fc6d.